ta/miss/ta_entry.c

```diff
@@ -0,0 +1,77 @@
+/*
```

Comment:
Why not `#include "../sims/ta_entry.c"`? Although this file is not exactly the same, I see no reason to duplicate code. `sims` supports more commands, but who can do more can do less.

Reply:
@omihalac have you tried my suggestion? Does it work?